urbanairship-chat-react-native/android/src/main/java/com/urbanairship/reactnative/chat/AirshipChatModule.java

```diff
@@ -51,6 +53,23 @@ public String getName() {
         return "AirshipChatModule";
     }
 
+    @ReactMethod
+    public void addOpenChatListener() {
```

Comment:
This should be added when the plugin is created instead of binding it to a method.